grammar.js

```diff
@@ -1329,7 +1394,7 @@ module.exports = grammar({
             ),
           ),
         ),
-        $.ordered_columns,
+        alias($.index_fields, $.ordered_columns)
```

Comment:
I think we can just change this to be `index_fields`

```suggestion
        $.index_fields
```